Lists/List/List.css

```diff
@@ -0,0 +1,52 @@
+@import "../../utility/component.css";
+
+.notification{
+    width: 100%
+}
+
+.Notification-card{
+  width: auto;
+  max-width: 100%;
+  height: auto;
+  background-color: var(--white);
+  box-shadow: 5px 5px 20px 0 rgb(0 0 0 / 20%);
+  border-radius:15px 0px 0px 5px;
+
+  display: flex;
+  flex-direction:row;
```

Comment:
You can make a class for flex and use

Reply:
I already have a flex utility class. but I don't want to use it here. 